src/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+set(HEADERS
+  binning.h
+  comm.h
+  examinimd.h
+  force.h
+  input.h
+  integrator.h
+  integrator_nve.h
+  math_extra.h
+  modules_binning.h
+  modules_comm.h
+  modules_force.h
+  modules_integrator.h
+  modules_neighbor.h
+  modules_property.h
+  neighbor.h
+  property_kine.h
+  property_pote.h
+  property_temperature.h
+  system.h
+  types.h)
+
+set(SOURCES
+  binning.cpp
+  comm.cpp
+  examinimd.cpp
+  force.cpp
+  input.cpp
+  integrator.cpp
+  integrator_nve.cpp
+  neighbor.cpp
+  property_kine.cpp
+  property_pote.cpp
+  property_temperature.cpp
+  system.cpp)
+
+add_executable(ExaMiniMD main.cpp ${SOURCES} ${HEADERS})
+
+set(SUBDIRECTORIES
+  binning_types
+  comm_types
+  force_types
+  neighbor_types)
+
+foreach(SUBDIR ${SUBDIRECTORIES})
+  add_subdirectory(${SUBDIR})
+endforeach()
+
+if (USE_CUDA)
+  set_target_properties(ExaMiniMD PROPERTIES LANGUAGE CUDA)
+endif()
+
+target_include_directories(ExaMiniMD PRIVATE ${Kokkos_DIR} ${CMAKE_CURRENT_SOURCE_DIR} ${SUBDIRECTORIES})
+
+set(EXAMINIMD_LIBS
```

Comment:
Meh, just use `target_link_libraries()` two times instead of making a special variable like `EXAMINIMD_LIBS`.

Reply:
On a second thought just use:
```
target_link_libraries(ExaMiniMD PRIVATE $<$<BOOL:${USE_MPI}>:MPI::MPI_CXX> Kokkos::kokkos)
```

Reply:
I tried using `target_link_libraries` twice, and CMake was having none of it, which is why I created the variable.

Reply:
51cd06621d17b6a165f9a3545355ce8a9bcd01f8